inc/Remote/XmlRpcServer.php

```diff
@@ -37,8 +37,8 @@ public function serve($data = false)
         if (
             !isset($_SERVER['CONTENT_TYPE']) ||
             (
-                strtolower($_SERVER['CONTENT_TYPE']) !== 'text/xml' &&
-                strtolower($_SERVER['CONTENT_TYPE']) !== 'application/xml'
+                str_contains(strtolower($_SERVER['CONTENT_TYPE']), 'text/xml') &&
+                str_contains(strtolower($_SERVER['CONTENT_TYPE']), 'application/xml')
```

Comment:
I think it would be better to split off the charset and then do a strict comparison. Even though requests with something like `myapplication/xml-is-stupid` are unlikely and would not really be a problem, I still think they should be rejected.

Reply:
Commit 17479e6b0e324af0256e0f4e71387f82ed0b9b69 looks for the strict string no matter how many items are sent in the content type